packages/ERTP/package.json

```diff
@@ -37,6 +37,7 @@
     "@agoric/evaluate": "^2.1.3",
     "@agoric/eventual-send": "^0.4.5",
     "@agoric/harden": "^0.0.4",
+    "@agoric/layer-cake": "https://github.com/DavidBruant/layer-cake.git#generator-less",
```

Comment:
This is temporary and to be fixed before merging

Reply:
Yup, makes sense